frontend_lib/src/component/Timeline/Timeline.jsx

```diff
@@ -256,6 +264,20 @@ export class Timeline extends React.Component {
             />
           </div>
         )}
+
+        {props.loggedUser.userRoleIdInWorkspace >= ROLE.contributor.id && props.showParticipateButton && (
+          <Link
+            style={{
+              '--primaryColor': props.customColor,
+              '--primaryDarkColor': darkenColor(props.customColor)
```

Comment:
can you give it the css classes instead?

Reply:
It's linked to the app type color, we do not have a css class for that. I'll rename the variables (it was inspired on the button type - primary) to `appTypeColor`.